docs/content/node-operation/db-encryption-existing-operator.mdx

```diff
@@ -0,0 +1,55 @@
+---
+title: Database Encryption for Existing Node Operators
+sidebar_title: Database Encryption for Existing Node Operators
+description: Instructions for existing Node Operators to follow to create a machine account for their collection or consensus nodes.
+---
+
+In Mainnet14, the DKG (distributed key generation) is turned on, requiring storage of 
+dynamically generated confidential data (random beacon keys). These are stored in a
+separate database which is new with the Mainnet14 release.
+
+All node operators joining after Mainnet14 will generate encryption keys for this database
+through the node bootstrapping and staking process. We strongly recommend all node operators
+(especially consensus node operators) generate an encryption key for this database. This
+guide demonstrates how to enable encryption for this database for existing operators.
+
+## Downloading Bootstrap Utility
+
+<Callout type="warning">
+  If you have downloaded the bootstrapping kit previously, ensure that you do
+  this step again to get the latest copy of the bootstrapping kit since there
+  have been significant changes to it.
+</Callout>
+
+Follow the instructions [here](/node-operation/node-bootstrap/#download-the-bootstrapping-kit)
+to download the latest version of the bootstrapping kit, then return to this page.
+
+## Generate Database Encryption Key
+
+You will need to generate an encryption key for the database using the `bootstrap` utility.
+
+<Callout type="warning">
+
+  Ensure you run the following commands on the machine you use to run your node software.
```

Comment:
Will the script complain if the directory passed with the `-o` option not the boostrap directory. On a call with BisonTrail yesterday, they told use that their key generation process is kept separate from the node and for the machine account key generation they had to modify the script to not fail if the node id file is not found.

Reply:
No it won't work  -- this works the same way as the machine account generation.

Reply:
Do you think we should change the script? The idea behind that requirement is that the script also moves the file in the correct location. 

I just tested, and another workaround they could use is to create a `node-id` file in the environment they use for key generation, then run the script.
```
mkdir fakebootstrap
mkdir fakebootstrap/public-root-information
echo $NODEID > fakebootstrap/public-root-information/node-id
./boot-tools/bootstrap db-encryption-key -o fakebootstrap
```